armi/interfaces.py

```diff
@@ -153,6 +153,11 @@ def __init__(self, r, cs):
         self.cs = settings.getMasterCs() if cs is None else cs
         self.r = r
         self.o = r.o if r else None
+        # default interface variables used for tight coupling
+        self.tightCouplingTolerance = 1e-4
+        self.tightCouplingVariables = None
+        self.tightCouplingConvergeOn = None
+        self.tightCouplingOldValue = None
```

Comment:
When we refer to `oldValue` i guess we are trying to say "previous" iteration value? What if this was a dictionary that was appended to and then on convergence was cleared?

Reply:
> When we refer to oldValue i guess we are trying to say "previous" iteration value?

Correct. 

> What if this was a dictionary that was appended to and then on convergence was cleared?

Dictionaries are not supported at this time. Just lists and floats.

Reply:
Sorry, meant list

Reply:
Make these `TIGHT_COUPLING_PARAMS`, and `TIGHT_COUPLING_COVERAGE_PARAMS` or something and then add a detailed comment here and in the docstring that explains what these are and how a user may add onto these for developments.